elastic/kibana
# [Entity Analytics][Entity Store] Refactor enablement UI

## Summary

This PR reworks the client side enablement flow for the Entity Store. It's the final piece for the work tracked in https://github.com/elastic/security-team/issues/10846, https://github.com/elastic/security-team/issues/10847 and https://github.com/elastic/security-team/issues/10947

https://github.com/user-attachments/assets/bb919c3c-b8dc-4e6b-a14b-4d413f8da13f

## How to test

Optionally 

On a fresh kibana and es cluster instance:
1. Load up some entity analytics data via the https://github.com/elastic/security-documents-generator
    * Running `yarn start entity-resolution-demo --mini` is enough 
1. Navigate to `Security > Dashboards > Entity Analytics`
3. Click the `Enable` entity store button
4. Both Risk Score and Entity Store toggles should be checked.
   * This state represents the engines we will install, _NOT_ the current state
5. Click `Enable`
6. The modal should close and Risk Scoring should initialize first
7. As soon as risk score initialization finished, the entity store initialization should start
   * Risk score related panels should show up while the store is initializing
8. Finally, the Entities List panel should appear   


##
 
- [x] added cypress tests to verify the correct enablement flow
